test/development/acceptance-app/ReactRefreshLogBox.test.ts

```diff
@@ -464,7 +464,18 @@ describe('ReactRefreshLogBox app', () => {
          "label": "Build Error",
          "source": "./index.module.css
        Parsing css source code failed
-       Selector is not pure (pure selectors must contain at least one local class or id), (lightningcss, Selector(button, specificity = 0x1))",
+       Selector is not pure (pure selectors must contain at least one local class or id), (lightningcss, Selector(button, specificity = 0x1))
+       Example import traces:
+         #1:
+           ./index.module.css [app-client]
+           ./index.js [app-client]
+           ./app/page.js [app-client]
+           ./app/page.js [app-rsc] [entrypoint]
+         #2:
+           ./index.module.css [app-ssr]
+           ./index.js [app-ssr]
+           ./app/page.js [app-ssr]
+           ./app/page.js [app-rsc] [entrypoint]",
```

Comment:
```suggestion
       Example import traces:
         app-client:
           ./index.module.css
           ./index.js
           ./app/page.js
           ./app/page.js [app-rsc]
         app-ssr:
           ./index.module.css
           ./index.js
           ./app/page.js
           ./app/page.js [app-rsc]",
```

Instead of the a number, show the layer. You can also avoid repeating the layer.

Reply:
there are cases where the layer changes in the trace (see test/development/sass-error/index.test.ts snapshots)

so we would need to handle when the layer changes.  so we could only display layer titles when they are different from the 'leaf layer', but then perhaps that is also ambiguous since some idents have no layers...